docs/dev/testing.md

```diff
@@ -73,5 +73,5 @@ b) you can then install the version using pip from git:
 
 Example:
 ```
-pip install git+https://github.com/pubpub-zz/pypdf.git@iss2200
+pip3 install git+https://github.com/pubpub-zz/pypdf.git@iss2200
```

Comment:
I disagree with this: Every Python user should either use `python-is-python3` (or similar) or a (dedicated) virtual environment defaulting to Python 3 nowadays. These docs are mostly for developers which work with the recent *pypdf* versions, where Python 2 is not supported anyway.